CRM/Utils/File.php

```diff
@@ -324,7 +327,7 @@ public static function runSqlQuery($dsn, $queryString, $prefix = NULL, $dieOnErr
         }
         catch (Exception $e) {
           if ($dieOnErrors) {
-            die("Cannot execute $query: " . $e->getMessage());
+            throw new CRM_Core_Exception("Cannot execute $query: " . $e->getMessage());
```

Comment:
`$dieOnErrors` seems a bit of a misnomer now

Reply:
Not sure why anyone would want to `echo` here either TBF.